### PR TITLE
chore: remove legacy display currency converter

### PR DESCRIPTION
**Pull request by vindard**
## Description

This PR removes remaining instances of `DisplayCurrencyConverter` and then removes `DisplayCurrencyConverter` implementation.

### CODE REVIEW

File: test/mocks/get-current-price.ts [hunk 1]
```diff
@@ -21,3 +24,12 @@ export const getCurrentUsdCentPrice = ({
...
 }
+
+export const getCurrentPriceAsPriceRatio = async ({
+  currency,
+}: GetCurrentSatPriceArgs): Promise<PriceRatio | PriceServiceError> => {
+  const price = await getCurrentSatPrice({ currency })
+  if (price instanceof Error) return price
+
+  return toPriceRatio(price.price * CENTS_PER_USD)
+}
```
Comment (dolcalmi) [inline test/mocks/get-current-price.ts:35]: were you getting an error without this? in theory if code did not change this is not necessary

File: src/domain/fiat/display-currency.ts [hunk 2]
```diff
@@ -1,29 +1,11 @@
 export const CENTS_PER_USD = 100
 
+export const usdMinorToMajorUnit = (amount: number | bigint) =>
+  Number((Number(amount) / CENTS_PER_USD).toFixed(2))
+
 export const toDisplayCurrencyBaseAmount = (amount: number) =>
   amount as DisplayCurrencyBaseAmount
 
-export const DisplayCurrencyConverter = (
```
Comment (dolcalmi) [inline src/domain/fiat/display-currency.ts:6]: will we remove/rename `NewDisplayCurrencyConverter` in a different PR?

**Review by dolcalmi — CHANGES_REQUESTED** [review]
looks good, just minor questions

File: test/mocks/get-current-price.ts (hunk 1 above)
Comment (vindard) [reply test/mocks/get-current-price.ts:35]: Yup tests fail with "undefined" function for that method

**Review by dolcalmi — APPROVED** [review]

**Pull request merged**